lib/src/drawings.dart

```diff
@@ -81,17 +88,23 @@ class TextDrawing extends GYWDrawing {
   final int? maxLines;
 
   /// Creates a text element.
-  const TextDrawing({
+  TextDrawing({
```

Comment:
Why is it not `const` anymore ?

Reply:
I don't remember why I removed it. I think I was testing some things before ending up with the current solution and I forgot to put it back.